src/main/webapp/app/exercises/programming/manage/update/update-components/custom-build-plans/programming-exercise-docker-image/programming-exercise-docker-image.component.html

```diff
@@ -11,6 +11,6 @@
         name="dockerImage"
         id="field_dockerImage"
         [(ngModel)]="dockerImage"
-        (ngModelChange)="dockerImageChange.emit($event)"
+        (ngModelChange)="dockerImage = $event.trim(); dockerImageChange.emit(dockerImage)"
```

Comment:
Can you move this into a method in the template? Imo we should only have method-delegation in html templates.

Reply:
This was suggested by ChatGPT :-) I also trim it in the corresponding setter, so we could probably remove it here again